.gitignore

```diff
@@ -1,92 +1,2 @@
-build/
-bin/
-lib/
-Debug/
-PreRelease/
-Profile/
-Release/
-ipch/
-__pycache__/
-src/data
-debian/autoreconf*
-debian/files
-debian/*.debhelper.log
-debian/*.substvars
-debian/pioneer*/
-autom4te.cache/
-codedoc/
-nd/Data/
-doxygen/html
-doxygen/latex
-*.vcxproj.user
-*.user
-*.ncb
-*.suo
-*.sdf
-*.opensdf
-*.o
-*.a
-.deps
-/*.exe
-/*.dll
-/*.pdb
-/*.ilk
-Makefile
-Makefile.in
-.DS_Store
-*.swp
-shipstat.csv
-pioneer
```

Comment:
should we not ignore the output of a build?

Reply:
It should be in `build`.

Reply:
I have a binary pioneer in my root (that points to build/). It compiles & runs the game.
(Granted, I usually don't use it)